docs/package.json

```diff
@@ -65,6 +65,7 @@
     "canvas": "^2.11.2",
     "dotenv-safe": "^8.2.0",
     "eslint-config-next": "^13.5.5",
+    "glob": "^10.3.10",
```

Comment:
How is it used in docs? I think this is only used during build. I think it's a dev dependency here too

Reply:
Yeah it's just used in the one file `docs/src/utils/getContentPaths.ts`, did you mean if it's only used in build we don't need to declare it here?